SpatialGDK/Source/Private/Interop/SpatialPlayerSpawner.cpp

```diff
@@ -62,6 +62,12 @@ void USpatialPlayerSpawner::ReceivePlayerSpawnResponse(Worker_CommandResponseOp&
 	{
 		UE_LOG(LogSpatialPlayerSpawner, Display, TEXT("Player spawned sucessfully"));
 	}
+#if WITH_EDITOR
```

Comment:
`WITH_EDITOR` is a preprocessor macro that turns on for editor builds, so this code will be active for editor runs with `-game` or `-server` too, including local deployments with managed workers. Mainly a technicality, but wanted to make sure you're aware.

Reply:
We likely also need a fix for this in cooked workers, in case they're overloaded. That might be lower priority though, will check whether it happens in Scavengers cooked clients.